### PR TITLE
Send mobile-purchases alarms to Growth and Value teams

### PR DESCRIPTION
**Pull request by tomrf1**
For now we want mobile-purchases alarms to go to both teams

### CODE REVIEW

File: handlers/alarms-handler/src/alarmMappings.ts [hunk 1]
```diff
@@ -49,22 +54,28 @@ const teamToAppMappings: Record<Team, string[]> = {
...
-);
+const buildAppToTeamMappings = (): Record<string, Team[]> => {
+	const mappings: Record<string, Team[]> = {};
+
+	for (const [team, apps] of Object.entries(teamToAppMappings)) {
+		for (const app of apps) {
+			const teams = mappings[app] ?? [];
+			teams.push(team as Team);
+
+			mappings[app] = teams;
```
Comment (tjmw) [inline handlers/alarms-handler/src/alarmMappings.ts:65]: I wonder if this is any clearer? Maybe it makes the fact that we're mutating/pushing onto `mappings[app]` a bit more obvious?

```suggestion
			mappings[app] ||= [];
			mappings[app].push(team);
```

**Review by tjmw — APPROVED** [review]
Nice! 👍

File: handlers/alarms-handler/src/alarmMappings.ts (hunk 1 above)
Comment (tjmw) [reply handlers/alarms-handler/src/alarmMappings.ts:65]: Ah right yeah good point - final decision is yours!

Comment (tjmw) [reply handlers/alarms-handler/src/alarmMappings.ts:65]: There's also [`??=`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_assignment) which is maybe slightly more correct here.

Comment (tomrf1) [reply handlers/alarms-handler/src/alarmMappings.ts:65]: I'm starting to wonder if there are too many operators in the language now!

Comment (tomrf1) [reply handlers/alarms-handler/src/alarmMappings.ts:65]: typescript seems to need an intermediary value - when I use either of these operators, it doesn't believe that `mappings[app]` exists -

Screenshot 2024-05-21 at 14 40 39

**Pull request merged**